### ISSUE TITLE
House and Senate Bills introduced in the last 2 weeks are not available in the published bulk data files

### ISSUE THREAD

**Issue by danielmroberts** [usgpo/bulk-data#160]
The latest bill in the HR bulk data files is HR 8080 (4/18), while the latest bill introduced is actually HR 8242 (5/2).

https://www.govinfo.gov/bulkdata/BILLS/118/2/hr
https://www.congress.gov/bill/118th-congress/house-bill/8242

Similarly, the latest Senate Bill in bulk data file has S4155 (4/17), while the latest introduced in the Senate is S4264 (5/2).

https://www.govinfo.gov/bulkdata/BILLS/118/2/s
https://www.congress.gov/bill/118th-congress/senate-bill/4262

Resolutions do not appear to have the same issue.

**Comment by jonquandt** [comment]
@danielmroberts  - I see HR 8076 is on the bulkdata site as of 5/4/2024
https://www.govinfo.gov/bulkdata/BILLS/118/2/hr/BILLS-118hr8076ih.xml

Similarly, [BILLS-118s4078is.xml](https://www.govinfo.gov/bulkdata/BILLS/118/2/s/BILLS-118s4078is.xml) is available as of Monday

To confirm, you are pulling from the bulkdata site and not the API, correct?
https://api.govinfo.gov/collections/BILLS/2024-05-01T00:00:00Z?offsetMark=*&pageSize=100&api_key=DEMO_KEY 

Note that there is often a difference between the availability of BILLSTATUS information and the actual bill text becoming available due to variability in the number of bill texts being provided to GPO and production schedules.

**Comment by danielmroberts** [comment]
@jonquandt Right. Those bills are available and receiving updates from what I can tell. However, the few hundreds of new bills introduced in the last couple weeks are not available.

For example, HR 8242 and S 4262.

**Comment by jonquandt** [comment]
Sorry, I should have been more clear - when there are a large number of bill texts submitted to GPO by the House and Senate, it can sometimes lead to delays in them getting posted to GovInfo. I am checking to see if there are any updates for when more recent new content will be available.

**Comment by danielmroberts** [comment]
@jonquandt Do you have an estimate on when this issue will be resolved?

**Comment by jonquandt** [comment]
Unfortunately, I do not at this time.

**Comment by danielmroberts** [comment]
Ok. We are looking at some other options for data until then. Thanks.
